{"uid":"a79fb01b2539f4ea","name":"test_create_user_with_invalid_user_name_length_or_characters_returns_400","fullName":"test.users.test_post_user#test_create_user_with_invalid_user_name_length_or_characters_returns_400","historyId":"e76becc2119ab24453c6b5a7feeb7101","time":{"start":1726088764100,"stop":1726088764239,"duration":139},"status":"failed","statusMessage":"AssertionError: Expected status code 400, but got 403. Response text:","statusTrace":"@pytest.mark.createuser\n    def test_create_user_with_invalid_user_name_length_or_characters_returns_400():\n        url = EndpointUser.get_base_user()\n        data = create_user_data(user_name=\"null\")\n        response = UserService.create_user(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/users/test_post_user.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [403]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 403. Response text:\n\ncore/assertions/status_code_assertion.py:37: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1726088720920,"stop":1726088720938,"duration":18},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Expected status code 400, but got 403. Response text:","statusTrace":"@pytest.mark.createuser\n    def test_create_user_with_invalid_user_name_length_or_characters_returns_400():\n        url = EndpointUser.get_base_user()\n        data = create_user_data(user_name=\"null\")\n        response = UserService.create_user(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/users/test_post_user.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [403]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 403. Response text:\n\ncore/assertions/status_code_assertion.py:37: AssertionError","steps":[],"attachments":[{"uid":"62d09e9e8ae5429c","name":"log","source":"62d09e9e8ae5429c.txt","type":"text/plain","size":1520},{"uid":"3897fdce9ab4d259","name":"stderr","source":"3897fdce9ab4d259.txt","type":"text/plain","size":1589}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"createuser"},{"name":"parentSuite","value":"test.users"},{"name":"suite","value":"test_post_user"},{"name":"host","value":"fv-az1532-294"},{"name":"thread","value":"2075-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.users.test_post_user"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["createuser"]},"source":"a79fb01b2539f4ea.json","parameterValues":[]}